<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IGIDataSourceManagerWidget</class>
 <widget class="QWidget" name="IGIDataSourceManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkIGIDataSourcemanager</string>
  </property>
  <property name="toolTip">
   <string>Press to advance time automatically.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="m_SourceSelectComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_AddSourcePushButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_RemoveSourcePushButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="m_HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="m_ToolManagerToolsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Double-click on a row in the list to activate a preview/control GUI for that data source.
Untick the checkbox to freeze-frame a data source. Recording incoming data will continue in the background;
just Data Storage will not receive updates.</string>
     </property>
     <property name="title">
      <string>Sources</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="m_TableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>A</string>
         </property>
         <property name="toolTip">
          <string>Active
Click to freeze all sources.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lag</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="m_ToolManagerConsoleGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Console</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="m_ToolManagerConsole">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>318</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>niftk::IGIDataSourcePlaybackWidget</class>
   <extends>QWidget</extends>
   <header>niftkIGIDataSourcePlaybackWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
